src/dataModels/JSON.js

```diff
@@ -174,10 +174,10 @@ var JSON = DataModel.extend('dataModels.JSON', {
                     value = at ? value.substr(0, at) + sortString + value.substr(at) : sortString + value;
                 }
             } else { // must be filter row
-                var behavior = this.grid.behavior,
-                    filter;
-                value = behavior.getGlobalFilter && (filter = behavior.getGlobalFilter()) &&
-                    filter.getColumnFilterState && filter.getColumnFilterState(this.getFields()[x]) || '';
+                if (!this.filter.getColumnFilterState) {
```

Comment:
I know we are doing this to accomodate the current logic...
However, in the long run, this logic we've inherited seems too prescriptive. If they want to use filter they should specify it (as a opposed to us reading a defaults property), its cell renderer, and cell editor via some mechanism like `getCell` (since we don't have first class row objects due to perf reasons).